lang/cs/Org.Apache.REEF.Tang/Formats/AvroConfigurationSerializer.cs

```diff
@@ -51,7 +51,6 @@ public override TypeSerializationInfo ResolveType(Type type)
         }
     }
 
-    // TODO[REEF-842] Act on the obsoletes
```

Comment:
REEF-842 contains changes in multiple places. In this file, the obsolete call is replaced with this PR. But not other files.